trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py

```diff
@@ -53,11 +53,9 @@ def compressibility_drag_wing(state,settings,geometry):
     
     wing = geometry
     if wing.tag == 'main_wing':
```

Comment:
We should make this if isinstance(wing,main_wing):

Reply:
Done